packages/store/src/module.ts

```diff
@@ -69,29 +53,11 @@ export class NgxsFeatureModule {
     factory: StateFactory,
     @Optional()
     @Inject(FEATURE_STATE_TOKEN)
-    states: any[][]
+    lazyStates: StateClass[][]
   ) {
-    // Since FEATURE_STATE_TOKEN is a multi token, we need to
-    // flatten it [[Feature1State, Feature2State], [Feature3State]]
-    const flattenedStates = ([] as any[]).concat(...states);
-
-    // add stores to the state graph and return their defaults
-    const results = factory.addAndReturnDefaults(flattenedStates);
-
-    const stateOperations = internalStateOperations.getRootStateOperations();
-    if (results) {
-      // get our current stream
-      const cur = stateOperations.getState();
-
-      // set the state to the current + new
-      stateOperations.setState({ ...cur, ...results.defaults });
-    }
-
-    stateOperations.dispatch(new UpdateState()).subscribe(() => {
-      if (results) {
-        factory.invokeInit(results.states);
-      }
-    });
+    const action: Type<unknown> = UpdateState;
+    const states: StateClass[] = [].concat(...(lazyStates as any));
```

Comment:
```suggestion
    // Since FEATURE_STATE_TOKEN is a multi token, we need to
    // flatten it [[Feature1State, Feature2State], [Feature3State]]
    const flattenedStates: StateClass[] = [].concat(...(lazyStates as any));
```
Please keep the name `flattenedStates` and the comments from above to preserve context of what is happening here.